### PR TITLE
Made the background pink

### PR DESCRIPTION
**Pull request by Strum355**
For LADWeed <3

Hi Reddit

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,11 +8,11 @@
     <head>
         <title>Netsoc Gamer Club</title>
     </head>
-    <body>
+    <body style="background: 'pink'">
```
Comment (mloc) [inline index.html:11]: not a nice pink

**Review by mloc — CHANGES_REQUESTED** [review]

**Comment by OisinA** [comment]
Noah hi

**Comment by OisinA** [comment]
thanks for acknowledging me

**Comment by ocanty** [comment]
uwu uwu

**Pull request merged**